weblab_cg/tests/chaste_test_utils.py

```diff
@@ -73,14 +73,14 @@ def get_file_lines(file_name, remove_comments=False):
 
 
 def write_file(file_name, file_contents):
-    """ Write Load a file into a list of lines
+    """ Write a file into a list of lines
 
     :param file_name: file name including path
     :param file_contents: a str with the contents of the file to be written
     """
     # Make sure the folder we are writing in exists
     try:
-        os.makedirs(os.path.dirname(file_name))
+        os.makedirs(os.path.dirname(file_name), exist_ok=True)

```

Comment:
You don't need the try-except with this change!

Also, I'm not sure the first line of the docstring is correct. Shouldn't it be something like "Write text to the given file"?